### DISCUSSION TITLE
Serverless functions plugin in Javascript/Go/Python

### DISCUSSION THREAD

**Discussion by anup-krai** [Kong/kong#10350] [Help] [answered]
We are exploring Kong [serverless-functions](https://docs.konghq.com/hub/kong-inc/serverless-functions/) plugin where we need to write code in Lua. Is there any plugin or example to achieve the same using plugin in other languages?

**Comment by pluveto** [comment, marked as answer]
Good idea. At the moment, for security reasons, the serverless sandbox only provides a few functions and cannot call another process. @dndx
